src/Orleans/Runtime/IGrainRuntime.cs

```diff
@@ -41,5 +42,7 @@ public interface IGrainRuntime
         void DeactivateOnIdle(Grain grain);
 
         void DelayDeactivation(Grain grain, TimeSpan timeSpan);
+
+        IStorage<TGrainState> GetStateStorageBridge<TGrainState>(Grain grain) where TGrainState : new();
```

Comment:
naming-wise this should be GetStorage, right? The bridge is the implementation

Reply:
I guess it should...